charts/pathfinder-rpc-server/templates/configmap.yaml

```diff
@@ -15,7 +15,7 @@ data:
   PATHFINDER_PROXY_UPSTREAM_SERVICE_ENDPOINTS:     {{ index .Values "pathfinder-proxy" "upstream" "service-endpoints" }}
   PATHFINDER_PROXY_UPSTREAM_HEALTH_ENDPOINTS:      {{ index .Values "pathfinder-proxy" "upstream" "health-endpoints" }}
   {{- end }}
-  {{- if .Values.pathfinder.enabled }}
+  {{- if index .Values "pathfinder" "enabled" }}
```

Comment:
The `index` notation for key is only needed, when those contain dashes ("`-`").

Reply:
removed back to camelcase then index is not needed

Reply:
Thanks! 55a0ad8 did the trick.